atd-vzv/src/views/nav/SideMapFooter.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import logo from "./coa_seal_transparent_white.png";
+
+const SideMapFooter = () => {
+  const handleFeedbackClick = () => {
```

Comment:
Thank you Mike! Just made the changes. Could you double-check for me?

Reply:
@RR216 that looks great to me, but I see there are a couple additional files in this diff since I last reviewed -> https://github.com/cityofaustin/atd-vz-data/pull/1256/files `atd-vzv/public/index.backup.html` (added) and `atd-vzv/public/index.html` (updated). 

I saw your commit message, and it seems that you may have run into a build issue? I'll approve as soon as we get that sorted out. I suspect that we can get away without those changes but I could be missing some context. Thanks!

Reply:
All good here - thanks for updating 🙏